conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosGovernanceOperations.scala

```diff
@@ -45,6 +48,28 @@ object TezosGovernanceOperations extends LazyLogging {
     */
   case class VoteRollsCounts(yay: Int, nay: Int, pass: Int)
 
+  object VoteRollsCounts {
+
+    /** a wrapper with all zeroes */
+    val zero = VoteRollsCounts(0, 0, 0)
+
+    /** defines substraction between counts, with the caveat
+      * that no result of the substraction can have negative counts,
+      * which would have no meaning
+      *
+      * @param subtrahend
+      * @param minuend
+      * @return the "zero-min" difference of the counts, element by element
+      */
+    def subtract(subtrahend: VoteRollsCounts, minuend: VoteRollsCounts): VoteRollsCounts = (subtrahend, minuend) match {
+      case (VoteRollsCounts(i1, j1, k1), VoteRollsCounts(i2, j2, k2)) =>
```

Comment:
can we get more meaningful names? even `y1`, `n1`, `p1` would be better than `i`,`j`,`k`.